sql/2019/20_HTTP_2/20_06.sql

```diff
@@ -0,0 +1,30 @@
+#standardSQL
+# 20.06 - Number of HTTPS sites not using HTTP/2 which return upgrade HTTP header containing h2
+CREATE TEMPORARY FUNCTION getUpgradeHeader(payload STRING)
+RETURNS STRING
+LANGUAGE js AS """
+  try {
+    var $ = JSON.parse(payload);
+    var headers = $.response.headers;
+    var st = headers.find(function(e) { 
+      return e['name'].toLowerCase() === 'upgrade'
+    });
+    return st['value'];
+  } catch (e) {
+    return '';
+  }
+""";
+
+SELECT 
+  _TABLE_SUFFIX AS client,
+  JSON_EXTRACT_SCALAR(payload, "$._is_base_page") AS is_base_page,  
```

Comment:
Ditto previous comments. Should `is_base_page` be a field in the output or a condition in the WHERE clause? Applies to the rest of these queries as well.